src/SME.AE.Infra/Persistencia/Repositorios/OutroServicoRepositorio.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Dapper;
+using Npgsql;
+using SME.AE.Aplicacao;
+using SME.AE.Aplicacao.Comum.Interfaces.Repositorios;
+using SME.AE.Comum;
+using SME.AE.Dominio.Entidades;
+using SME.AE.Infra.Persistencia.Repositorios;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SME.AE.Infra
+{
+    public class OutroServicoRepositorio : IOutroServicoRepositorio
+    {
+        private readonly VariaveisGlobaisOptions variaveisGlobaisOptions;
+        public OutroServicoRepositorio(VariaveisGlobaisOptions variaveisGlobaisOptions)
+        {
+            this.variaveisGlobaisOptions = variaveisGlobaisOptions ?? throw new ArgumentNullException(nameof(variaveisGlobaisOptions));
+        }
+        private NpgsqlConnection CriaConexao() => new NpgsqlConnection(variaveisGlobaisOptions.AEConnection);
+        public async Task<IEnumerable<OutroServicoDto>> Links()
+        {
+            try
+            {
+                using var conexao = CriaConexao();
+                conexao.Open();
+                var lista = await conexao.QueryAsync<OutroServicoDto>(@"
+                        select 
+	                        titulo,
+	                        descricao,
+	                        categoria,
+	                        urlsite,
+	                        icone,
+	                        destaque
+                        from OutroServico
+                        where ativo
+                        order by categoria,titulo");
+                conexao.Close();
+
+                return lista;
+            }
+            catch (Exception)
+            {
+                throw;
+            }
+        }
+
+        public async Task<IEnumerable<OutroServicoDto>> LinksPrioritarios()
+        {
+            try
+            {
+                using var conexao = CriaConexao();
+                conexao.Open();
+                var lista = await conexao.QueryAsync<OutroServicoDto>(@"
+                        select 
+	                        titulo,
+	                        descricao,
+	                        categoria,
+	                        urlsite,
+	                        icone,
+	                        destaque,ordem
+                        from OutroServico
+                        where ativo and destaque
+                        and ordem between 1 and 6
```

Comment:
porque ordem entre 1 e 6 ?

Reply:
é um critério de aceite da história que só pode exibir registros 6 registros, ai o  between 1 and 6 para caso alguém faça insert de um registro com ordem 0  ou maior que 6 ele não vai aparecer

Reply:
isso para os links em destaque